packages/docs/getting-started/running-a-validator-in-baklava.md

```diff
@@ -173,6 +190,46 @@ docker pull $CELO_IMAGE
 
 The `us.gcr.io/celo-org/geth:baklava` image contains the [genesis block](https://github.com/celo-org/celo-monorepo/tree/master/packages/celotool/genesis_baklava.json) in addition to the Celo Blockchain binary.
 
+#### Account Creation (For non-ReleaseGold)
+{% hint style="info" %}
+Please complete this section if you are new to validating on Celo.
+{% endhint %}
+
+If you're a new validator, then you are going to follow the non-ReleaseGold flow method for this step to create your account keys. 
+
+##### Account and Signer keys
+
+Running a Celo Validator node requires the management of several different keys, each with different privileges. Keys that need to be accessed frequently (e.g. for signing blocks) are at greater risk of being compromised, and thus have more limited permissions, while keys that need to be accessed infrequently (e.g. for locking CELO) are less onerous to store securely, and thus have more expansive permissions. Below is a summary of the various keys that are used in the Celo network, and a description of their permissions.
+
+| Name of the key        | Purpose                                                                                                                                                                                                                                                               |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Account key            | This is the key with the highest level of permissions, and is thus the most sensitive. It can be used to lock and unlock CELO, and authorize vote, validator, and attestation keys. Note that the account key also has all of the permissions of the other keys. |
+| Validator signer key   | This is the key that has permission to register and manage a Validator or Validator Group, and participate in BFT consensus.                                                                                                                                          |
+| Vote signer key        | This key can be used to vote in Validator elections and on-chain governance.                                                                                                                                                                                          |
+| Attestation signer key | This key is used to sign attestations in Celo's lightweight identity protocol.                                                                                                                                                                                        |
+
+Note that account and signer keys must be unique and may not be reused.
```

Comment:
To clarify, the Account key, Validator signer key, Vote signer key, and attestation signer key must all be unique?